bindings/cs/MongoDB.Crypt/CryptClientFactory.cs

```diff
@@ -36,7 +36,20 @@ public static CryptClient Create(CryptOptions options)
             kmsCredentials.SetCredentials(handle,status);
 
             // TODO - set logger
```

Comment:
I guess no logger since the driver doesn't log either?

Reply:
True, I'll remove the comment.